editor/components/provider/index.js

```diff
@@ -33,17 +33,48 @@ import store from '../../store';
 const DEFAULT_SETTINGS = {
 	alignWide: false,
 	colors: [
-		'#f78da7',
-		'#cf2e2e',
-		'#ff6900',
-		'#fcb900',
-		'#7bdcb5',
-		'#00d084',
-		'#8ed1fc',
-		'#0693e3',
-		'#eee',
-		'#abb8c3',
-		'#313131',
+		{	name: 'pale pink',
```

Comment:
Consistency: Property on own line. Same with next object. Remainder of objects are correct.